packages/Calendar/styles.js

```diff
@@ -0,0 +1,197 @@
+import styled from 'styled-components'
+
+const theme = {
+  colors: {
+    telusPurple: '#4B286D',
+    shark: '#2A2C2E',
+    shuttleGrey: '#54595F',
+    white: '#FFF',
+    gainsboro: '#D8D8D8',
+  },
+  fontWeights: {
+    thin: 300,
+    light: 400,
+    roman: 500,
+    medium: 700,
+  },
+  breakPoints: {
```

Comment:
These breakpoints can now be imported using `@tds/core-responsive@1.2.0` See the **media utility usage** section for a handy helper for generating media queries similar to the existing `sass-mq` usage.